open-craft/xblock-poll
# [BB-2984] Fix Poll XBlock broken export results feature

This PR removes the csv export feature in Poll XBlock. The CSV export feature is broken and the one on the Instructor Dashboard > Data Download works so we should remove the one in the XBlock since it is confusing.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-2984

~~**Discussions**:~~ 

**Dependencies**: None

**Screenshots**: 
![Screenshot from 2020-09-23 21-52-18](https://user-images.githubusercontent.com/1010244/94037371-1d5b0b00-fde7-11ea-9ad9-eb414d216f04.png)


~~**Sandbox URL**:~~

**Merge deadline**: "None"

**Testing instructions**:

1. Install xblock-poll in a lms and studio
2. Add a poll from studio
3. As a staff user visit lms poll page. Ensure there are no ``Export results to CSV`` and ``Download CSV`` button.

**Author notes and concerns**:
N/A

**Reviewers**

~~**Settings**~~
